### ISSUE TITLE
fix pilot importer to process first row

### ISSUE THREAD

**Issue by mjmdavis** [fastlane/fastlane#4912]
### New Issue Checklist

- [X] Updated fastlane to the latest version
- [X] I have read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)

### Issue Description
The pilot importer does not process the first row of the provided csv. I assume that it expects the first row has header information in it but the example linked in the [documentation](https://github.com/fastlane/fastlane/tree/master/pilot#import-testers) does not.

##### Complete output when running fastlane, including the stack trace and command used
```bash
pilot import -c ./tester_import.csv --verbose
Login to iTunes Connect (mike@mjmdavis.com)
Login successful
Successfully imported 0 testers from ./tester_import.csv
```

### Configuration Files
csv file:
```csv
John,Appleseed,appleseed_john@mac.com
```

### Environment

fastlane version (run `fastlane -v`):
fastlane 1.91.0
pilot 1.8.0

Do you use bundler to execute fastlane (i.e. `bundle exec fastlane`)?
no

Do you use a Ruby environment manager (e.g. `chruby`, `rbenv`, `rvm`)?
yes

**Comment by TKBurner** [comment]
Hi @mjmdavis Thanks for following up. This is expected behavior right now, we'll decide which way to go (updated docs or merging your PR) and update on your PR. Thanks!

**Comment by mjmdavis** [comment]
Shouldn't the issue stay open until the fix is made?

I'm curious to know what your process around this is.

**Comment by KrauseFx** [comment]
@mjmdavis Yeah, we can re-open, but make sure that your PR body contains `Fixes #[number]`, so that this issue gets auto-closed once the PR is merged 👍

**Comment by mjmdavis** [comment]
Updated the PR to include `Fixes #<issue_number>` 😄
Danke!

**Comment by TKBurner** [comment]
Thanks @mjmdavis! We'll work with you on this. 👍

**Comment by KrauseFx** [comment]
Fixed with https://github.com/fastlane/fastlane/pull/4119 🚀
